pkg/controller/status/status.go

```diff
@@ -415,11 +427,18 @@ func (m *statusManager) syncState() bool {
 	return m.explicitDegradedReason != ""
 }
 
+// isCRQueried returns true if corresponding CR has been queried
+func (m *statusManager) isCRQueried() bool {
```

Comment:
Similar to my comment below, I think this should be something like "isInitialized()", since the status manager code is meant to be more-or-less agnostic to the logic that is driving it.